workflow_test.go

```diff
@@ -58,7 +58,7 @@ func (s status) String() string {
 	case StatusInitiated:
 		return "Initiated"
 	case StatusProfileCreated:
-		return "Name Created"
+		return "name Created"
```

Comment:
Was this one intended to be changed? 

```suggestion
		return "Name Created"
```

Reply:
ah good catch. Not intended.